x/gov/client/cli/query.go

```diff
@@ -242,15 +242,16 @@ $ %s query gov vote 1 cosmos1skjwj5whet0lpe65qaq4rpq03hjxlwd9nf39lk
 
 // GetCmdQueryVotes implements the command to query for proposal votes.
 func GetCmdQueryVotes(queryRoute string, cdc *codec.Codec) *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "votes [proposal-id]",
 		Args:  cobra.ExactArgs(1),
 		Short: "Query votes on a proposal",
 		Long: strings.TrimSpace(
 			fmt.Sprintf(`Query vote details for a single proposal by its identifier.
 
 Example:
-$ %s query gov votes 1
+$ %[1]s query gov votes 1
```

Comment:
Undo the `%[1]s`.

Reply:
why? it is used on this line and the next one. do you think it is better to repeat parameter?

Reply:
Nvm, this is fine 👍